Gems/Multiplayer/Code/Include/Multiplayer/Components/NetworkCharacterComponent.h

```diff
@@ -53,6 +55,9 @@ namespace Multiplayer
         void OnActivate(Multiplayer::EntityIsMigrating entityIsMigrating) override;
         void OnDeactivate(Multiplayer::EntityIsMigrating entityIsMigrating) override;
 
+    protected:
+        void OnCharacterActivated() override;
```

Comment:
Should we have an OnCharacterDeactiveated() call?

Reply:
I answered this in a separate code comment below.